### PR TITLE
Added app styles

### PR DESCRIPTION
**Pull request by Asiimwe-Mark**
I added the necessary styles to the app following the project requirements below;

Add any missing JSX elements to your project (without adding extra functionality)
Your final design should match Zeplin's design (this includes all elements + the same styling)
You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/App.css
Comment (aamirkhan2478) [inline src/App.css:1]: - [ ] Kindly use microverse provided design `font-family` in your case you didn't use Zaplin design font-family so kindly fix it :slightly_smiling_face: 

Zaplin Font-family
- Montserrat
- RobotoSlab

File: src/components/nav.js [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { Link } from 'react-router-dom';
...
-    <span>Bookstore CMS</span>
+    <span className="cms">Bookstore CMS</span>
     <nav>
-      <Link to="/">Book</Link>
-      <Link to="/categories">Categories</Link>
+      <Link className="nav-book" to="/">Book</Link>
+      <Link className="nav-category" to="/categories">Categories</Link>
     </nav>
   </div>
 );
```
Comment (aamirkhan2478) [inline src/components/nav.js]: - [ ] Please stick to the Zaplin design you missed the user icon on the right side if you check the Zaplin design there is one user icon on the right side but in your case, you missed this icon so kindly add this icon as well :slightly_smiling_face: 
- [ ] If you check Zaplin design's navbar you will see there is border on the bottom side but in your case this border is not there so kindly fix this issue as well :+1: 

**Your Design**
Screenshot from 2023-08-24 00-31-49

**Zaplin Design**
Screenshot from 2023-08-24 00-30-18

File: src/components/BookCard.js [hunk 2]
```diff
@@ -13,28 +13,29 @@ const BookCard = ({ books }) => {
...
             <div className="Book-btns">
-              <button type="button">Comments</button>
-              <div className="divide" />
-              <button onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
-              <div className="divide" />
-              <button type="button">Edit</button>
+              <button className="comment" type="button">Comments</button>
+              <div className="Line-2" />
+              <button className="remove" onClick={() => { removeBook(book.item_id); }} type="button">Remove</button>
+              <div className="Line-2" />
```
Comment (aamirkhan2478) [inline src/components/BookCard.js:23]: - [ ] Please stick to the Zaplin design if you check dividers between comments and remove buttons the width is not matched with Zaplin Design so kindly fix this issue as well  :slightly_smiling_face: 

**Your Design**
Screenshot from 2023-08-24 00-48-07

**Zaplin Design**
Screenshot from 2023-08-24 00-47-41

File: src/components/BookCard.js [hunk 3]
```diff
@@ -13,28 +13,29 @@ const BookCard = ({ books }) => {
...
             <div className="progress" />
             <div className="progressInfo">
-              <p>
+              <p className="percentage">
                 64%
               </p>
-              <p>Completed</p>
+              <p className="completed">Completed</p>
             </div>
+            <div className="Line-3" />
```
Comment (aamirkhan2478) [inline src/components/BookCard.js:35]: - [ ] As I mentioned above :point_up: please stick to this Zaplin design if you check dividers between the percentage and progress button the width is not matched with Zaplin Design so kindly fix this issue as well 🙂

**Your Design**
Screenshot from 2023-08-24 00-51-25

**Zaplin Design**
Screenshot from 2023-08-24 00-51-35

File: src/components/Book.js [hunk 4]
```diff
@@ -4,6 +4,7 @@ import AddBook from './AddBook';
 const Book = () => (
   <div className="Books">
     <BookList />
+    <div className="Line" />
```
Comment (aamirkhan2478) [inline src/components/Book.js:7]: - [ ] As I mentioned above ☝️ please stick to this Zaplin design if you check the divider the height is not matched with Zaplin Design so kindly fix this issue as well 🙂

**Your Design**
Screenshot from 2023-08-24 00-54-50

**Zaplin Design**
Screenshot from 2023-08-24 00-55-25

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @As1imwe-Mark, 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort
✔️ Your design is good.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.css [hunk 5]
```diff
@@ -1,5 +1,313 @@
...
+  width: 4.563rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+}
+
+.remove {
+  width: 3.375rem;
+  height: 1.188rem;
+  font-family: 'Roboto Slab', serif;
+}
```
Comment (1GORDON) [inline src/App.css:116]: Great job @As1imwe-Mark 🤝

- [ ] [Optional] As stated in the project requirements, Ensure you do not repeat the same rule for multiple selectors for example `Roboto Slab', serif`. Kindly create a single class and reuse this class where necessary. You may take a look at [this file](https://speckyboy.com/good-bad-css-practices/) for more guidance.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @As1imwe-Mark ✋

Your project is complete!  There is nothing else to say other than... It's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Replicated the Zeplin design ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**